File: task-03/Javascript/subtask-2.js
const fs = require('fs');

function fileReadAndWrite() {
    const inputPath = "/home/aakash/amfoss-tasks/task-03/Javascript/input.txt";
    const outputPath = "/home/aakash/amfoss-tasks/task-03/Javascript/output.txt";

    fs.readFile(inputPath, 'utf8', (err, data) => {
        if (err) {
            console.error(err);
            return;
        }
        fs.writeFile(outputPath, data, (err) => {
            if (err) {
                console.error(err);
            }
        });
    });
}

fileReadAndWrite();

File: task-03/Rust/subtask-2.rs
use std::fs;
use std::io::Write;

fn file_read_and_write() -> std::io::Result<()> {
    let input_path = "/home/aakash/amfoss-tasks/task-03/Rust/input.txt";
    let output_path = "/home/aakash/amfoss-tasks/task-03/Rust/output.txt";

    let data = fs::read_to_string(input_path)?;

    fs::write(output_path, data)?;

    Ok(())
}

fn main() {
    if let Err(e) = file_read_and_write() {
        eprintln!("Error: {}", e);
    }
}